app/views/hyrax/base/_workflow_actions.html.erb

```diff
@@ -1,11 +1,11 @@
-<div id="workflow_controls" class="panel panel-default workflow-affix">
```

Comment:
workflow-affix is bound to code in `app/assets/javascripts/hyrax/workflow_actions_affix.js`.  can that script be removed now?

Reply:
Removed.